frontend/lib/parsing/parsing-queries.cpp

```diff
@@ -388,21 +388,26 @@ void countTokens(Context* context, UniqueString path, ParserStats* parseStats) {
   }
 }
 
+static const BuilderResult*
+builderResultPtr(Context* context, ID id, UniqueString& pathOut) {
```

Comment:
I think the Dyno convention has been to name these functions with 'OrNull'. E.g.:

```
  const ResolvedExpression* byAstOrNull(const uast::AstNode* ast) const {
```

So then:
```suggestion
builderResultPtrOrNull(Context* context, ID id, UniqueString& pathOut) {
```